### PR TITLE
Add RPCS3 Linux Solution

### PR DESCRIPTION
**Pull request by irrdkwhattoput**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +205,14 @@ To undo the changes from Zadig, you have to rollback the driver:
 2. Double-click `LEGO READER V2.10`.
 3. Switch to the `Driver` tab, click `Previous Driver`, select the first option and click yes.
 
+**Solution for Linux systems**
+
+In order to fix this you will need to add a custom udev rule in your system so the software can comunicate with the game, the udev rule is in the root of the server folder. (thanks to wof for sharing the rule)
+
+To add the rule simply just move the `99-dimensions.rules` file to `/etc/udev/rules.d/` (a reboot may be required)
```
Comment (Berny23) [inline README.md:212]:
```suggestion
To add the rule, simply move the `99-dimensions.rules` file to `/etc/udev/rules.d/` (a reboot may be required).
```

File: README.md [hunk 2]
```diff
@@ -205,6 +205,14 @@ To undo the changes from Zadig, you have to rollback the driver:
 2. Double-click `LEGO READER V2.10`.
 3. Switch to the `Driver` tab, click `Previous Driver`, select the first option and click yes.
 
+**Solution for Linux systems**
+
+In order to fix this you will need to add a custom udev rule in your system so the software can comunicate with the game, the udev rule is in the root of the server folder. (thanks to wof for sharing the rule)
```
Comment (Berny23) [inline README.md:210]:
```suggestion
In order to fix this, you will need to add a custom udev rule in your system, so the software can comunicate with the game. The udev rule is in the root of the server folder. (thanks to wof for sharing the rule).
```

**Review by Berny23 — CHANGES_REQUESTED** [review]
Thanks for your contribution!

Just some small grammatical improvements.

**Comment by irrdkwhattoput** [comment]
Fixed, thanks for telling me. 👍

**Review by irrdkwhattoput — COMMENTED** [review]
Fixed all grammar errors + some other improvements

**Pull request merged**